src/docker/v2/storage/blobs.rs

```diff
@@ -0,0 +1,114 @@
+/*
+   Copyright 2021 JFrog Ltd
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+pub use crate::artifacts_repository::hash_util::HashAlgorithm;
```

Comment:
again expose definition seems out of place

```suggestion 
use crate::artifacts_repository::hash_util::HashAlgorithm;
```